src/AppInstallerCLICore/ExecutionReporter.h

```diff
@@ -23,6 +23,18 @@ namespace AppInstaller::CLI::Execution
 {
 #define WINGET_OSTREAM_FORMAT_HRESULT(hr) "0x" << Logging::SetHRFormat << hr
 
+    // The level for the Output channel.
+    enum class ReporterLevel : uint32_t
+    {
+        None = 0x0,
+        Verbose = 0x1,
+        Info = 0x2,
+        Warning = 0x4,
+        Error = 0x8,
+        All = Verbose | Info | Warning | Error,
+    };
+    DEFINE_ENUM_FLAG_OPERATORS(ReporterLevel);

```

Comment:
If you previously had tried to place `DEFINE_ENUM_FLAG_OPERATORS` inside the `Reporter` struct alongside the enum, that was the issue.  It would need to be outside of the struct and reference a more qualified name, like: `DEFINE_ENUM_FLAG_OPERATORS(Reporter::Level)`.

Reply:
I believe we have existing usage [here](https://github.com/microsoft/winget-cli/blob/464e177f3b355ffef3d9f96fdebed9004407d181/src/AppInstallerRepositoryCore/Microsoft/Schema/ISQLiteIndex.h#L121) Does making the variables fully qualified help?

Reply:
Hmm. I thought I had tried that already, but perhaps I overlooked it. Will check

Reply:
Even making the variables fully qualified results in the same error . . .